src/screens/FileInput.js

```diff
@@ -101,12 +100,15 @@ export default () => (
             <Description>
               If an object is supplied, 'aggregageThreshold' indicates the
               maximum number of individual files to show. Above this, only a
-              single message describing the number of files will be shown.
+              single message describing the number of files will be shown. If an
+              object is supplied, 'max' indicates the maximum number of files
+              that can be added.
```

Comment:
"If an object supplied is redundant since that was stated at the beginning of the description.

```suggestion
              single message describing the number of files will be shown. 'max' 
              indicates the maximum number of files that can be added.
```